Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -1121,10 +1192,13 @@ void FitPeaks::fitSpectrumPeaks(
       // point)
       cost =
           fitIndividualPeak(wi, peak_fitter, expected_peak_pos, peak_window_i,
-                            observe_peak_params, peakfunction, bkgdfunction);
+                            observe_peak_width, peakfunction, bkgdfunction);
       if (cost < 1e7) { // assume it worked and save out the result
-        for (size_t i = 0; i < lastGoodPeakParameters[fit_index].size(); ++i)
+        for (size_t i = 0; i < lastGoodPeakParameters[fit_index].size(); ++i) {
```

Comment:
Also I don't think it matters, but it might be nice if the order of elements in lastGoodPeakParameters is the same as the peak list - i.e. indexed with peak_index not fit_index (again something I missed in the original PR)

Reply:
Done!